client/landing/stepper/declarative-flow/internals/steps-repository/import/style.scss

```diff
@@ -11,3 +11,16 @@
 		margin: auto;
 	}
 }
+
+@media ( max-width: 1000px ) {
```

Comment:
We should use the defined breakpoints mixins, such as `@include break-xlarage {}`.

There are constants:
`break-xlarge` => 1080px
`break-large` => 960px

Why `1000px`, is this something from design?

Reply:
It can be changed to `@include break-xlarge`. I will add a PR for that.